dev/io.openliberty.persistence.3.1.thirdparty/bnd.bnd

```diff
@@ -0,0 +1,132 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+# Eclipselink version info
+eclVersion=3.1.0
+eclFullVersion=${eclVersion}
+eclPackageVersion=3.1.0
+asmFullVersion=9.2.0
+
+Bundle-SymbolicName: io.openliberty.persistence.3.1.thirdparty
+Bundle-Description: EclipseLink JPA
+Bundle-Distributor: IBM
+Bundle-Activator: io.openliberty.persistence.internal.helper.EclipselinkPersistenceActivator
+Bundle-Name: EclipseLink JPA
+Premain-Class: org.eclipse.persistence.internal.jpa.deployment.JavaSECMPInitializerAgent
+Bundle-RequiredExecutionEnvironment: JavaSE-11
+Bundle-Vendor: Eclipse.org - EclipseLink Project
+Bundle-ManifestVersion: 2
+Specification-Version: ${eclVersion}
+
+javac.source: 11
+javac.target: 11
```

Comment:
`Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"` should also be added.